libraries/AP_HAL_ChibiOS/hwdef/FoxeerF405v2/README.md

```diff
@@ -86,6 +81,11 @@ The correct battery setting parameters are:
 
 The FoxeerF405v2 does not have a builtin compass, but you can attach an external compass using I2C on the SDA and SCL pads.
 
+## Control GPIO pins
```

Comment:
should also mention GPIO70 (VTX pwr control I think)...RELAY2....high = off I think

Reply:
We don't know that is what it is. Without confirmation from Foxeer I think its better to leave it out.

Reply:
I'll approve but we need to bug Foxeer to determine what GPIO70 actually does.....and then add it to the readme....if it is indeed the VTX power as I suspect, it will cause user issues down the line